R/epinow.R

```diff
@@ -406,8 +406,8 @@ regional_epinow <- function(reported_cases,
   regional_errors <- purrr::compact(regional_errors)
 
   if (length(regional_errors) != 0) {
-     message("Runtime errors caught: ")
-     print(regional_errors)
+     futile.logger::flog.info("Runtime errors caught: ")
```

Comment:
@seabbs should these be warnings?

message levels are typically:
error - Something went wrong badly enough that I'm stopping
warning - something went wrong but I can continue
info - progress reports
debug - developer messages, values, more fine grained progress reports
trace - very low level debug

Reply:
Yes potentially. I wasn't sure if the user would always want to know this and so have it at the highest level of message. 

Perhaps the fact something has gone wrong should be info and then the detail as a warning?

Reply:
I think it's a question of is it really an error or not - I assumed that it was. If it's expected behaviour and not something to worry about unless debugging then perhaps it can be downgraded to a debug message